### PR TITLE
#26: Move code into sub package db

### LINKED ISSUE #26: Move code into sub-package db
## Background
We want our db related libraries better organized.

## Feature
Move the library to `za.co.absa.db.balta` package path.

### PR DESCRIPTION
**Pull request by benedeki**
Closes #26

Release notes:
Moving to better organization of the database related libraries. They are all now grouped in `za.co.absa.db...`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
 # Balta
 
-Scala library to write Postgres DB code tests with
+Scala library to used for writing Postgres DB code tests.
```
Comment (lsulak) [inline README.md:3]:
```suggestion
Scala library to be used for writing Postgres DB code tests.
```

or

```suggestion
Balta makes testing DB entities from Scala easier. It's primarily focused on testing the behavior of [Postgres functions](https://www.postgresql.org/docs/current/xfunc.html).
```

**Review by lsulak — COMMENTED** [review]
Just quickly read the changes in my web browser, it looks good to me, although the wording in the README's beginning could be improved.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-Scala library to write Postgres DB code tests with
+Balta makes testing DB entities from Scala easier. It's primarily focused on testing the behavior of [Postgres functions](https://www.postgresql.org/docs/current/xfunc.html).
+
+---
+
+### Build Status
+
+[![Build](https://github.com/AbsaOSS/balta/workflows/Build/badge.svg)](https://github.com/AbsaOSS/balta/actions)
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.db/balta_2.12/badge.svg)](https://search.maven.org/search?q=g:za.co.absa.db.balta)
```
Comment (miroslavpojer) [inline README.md:11]: Why Maven central?

File: balta/src/main/scala/za/co/absa/db/balta/classes/DBTable.scala [hunk 3]
```diff
@@ -14,10 +14,12 @@
...
 
-package za.co.absa.balta.classes
+package za.co.absa.db.balta.classes
 
-import za.co.absa.balta.classes.setter.{AllowedParamTypes, Params, SetterFnc}
-import za.co.absa.balta.classes.setter.Params.NamedParams
+import za.co.absa.db.balta.classes.setter.{AllowedParamTypes, Params, SetterFnc}
+import za.co.absa.db.balta.classes.setter.Params.NamedParams
+import za.co.absa.db.balta.classes.setter.Params.NamedParams
+import za.co.absa.db.balta.classes.setter.{AllowedParamTypes, Params}
```
Comment (miroslavpojer) [inline balta/src/main/scala/za/co/absa/db/balta/classes/DBTable.scala:22]:
```suggestion
```

File: balta/src/main/scala/za/co/absa/db/balta/classes/DBFunction.scala [hunk 4]
```diff
@@ -14,10 +14,11 @@
...
  */
 
-package za.co.absa.balta.classes
+package za.co.absa.db.balta.classes
 
-import za.co.absa.balta.classes.DBFunction.{DBFunctionWithNamedParamsToo, DBFunctionWithPositionedParamsOnly, ParamsMap}
-import za.co.absa.balta.classes.setter.{AllowedParamTypes, SetterFnc}
+import DBFunction.{DBFunctionWithNamedParamsToo, DBFunctionWithPositionedParamsOnly, ParamsMap}
+import za.co.absa.db.balta.classes.setter.{AllowedParamTypes, SetterFnc}
+import za.co.absa.db.balta.classes.setter.{AllowedParamTypes, SetterFnc}
```
Comment (miroslavpojer) [inline balta/src/main/scala/za/co/absa/db/balta/classes/DBFunction.scala:21]:
```suggestion
```

**Review by miroslavpojer — COMMENTED** [review]
- [ ] pulled
- [ ] code review

File: README.md (hunk 2 above)
Comment (benedeki) [reply README.md:11]: Not sure what you mean by the question. It's a link to the Maven repository for the lib.

**Review by miroslavpojer — APPROVED** [review]

**Pull request merged**